Externals/hidapi/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+project(hidapi)
+
+set(SRCS)
+set(LIBS)
+
+if(APPLE)
+  list(APPEND SRCS mac/hid.c)
+elseif(MSVC)
+  list(APPEND SRCS windows/hid.c)
+else()
+  find_package(Libudev)
+  if(${CMAKE_SYSTEM_NAME} STREQUAL "Linux" AND LIBUDEV_FOUND)
+    list(APPEND SRCS linux/hid.c)
+    list(APPEND LIBS udev)
+  else()
+    list(APPEND SRCS libusb/hid.c)
+  endif()
+endif()
+
+add_library(hidapi ${SRCS})
+target_link_libraries(hidapi PRIVATE ${LIBS})
+target_include_directories(hidapi PUBLIC hidapi)
+
+add_library(Hidapi::Hidapi ALIAS hidapi)
```

Comment:
I'm using it here as a literal alias, since FindHIDAPI now also produces `Hidapi::Hidapi` and it's nice to have the names be the same.